### PR TITLE
Feature/view event

### PR DESCRIPTION
**Pull request by LeyaimJimenezSoftware**
User can see the events and can see the event information

### CODE REVIEW

File: navigation/MainTabNavigator.js [hunk 1]
```diff
@@ -13,9 +14,11 @@ import Icon from 'react-native-vector-icons/MaterialIcons'
 import IconCreate from 'react-native-vector-icons/Ionicons'
 import IconProfile from 'react-native-vector-icons/MaterialCommunityIcons'
 import { EditUser } from '../screens/EditUser'
+import { ShowEvent } from '../components/ShowEvent'
+import DateTime from '../screens/DateTime'
```
Comment (JosePeregrina) [inline navigation/MainTabNavigator.js]: Please, Divide the code. Shouldn't there are the stacks of navigation in the same file

File: components/EventCard.js [hunk 2]
```diff
@@ -1,16 +1,28 @@
-import React from 'react';
-import {StyleSheet ,View, Text , Image, ScrollView} from 'react-native';
+import React from 'react'
+import { StyleSheet ,View, Text , Image, Button } from 'react-native'
```
[the remark this answers is not in the document]
Comment (JosePeregrina) [reply components/EventCard.js]: Haha XD

File: components/ShowEvent.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    marginHorizontal: 20,
+    marginBottom: 5,
+    fontSize: 20,
+    color: 'black',
+  },
+  nameDateContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  }
+})
```
Comment (fernandoamz) [inline components/ShowEvent.js]: @LeyaimJimenezSoftware, you need to add an enter at the end of each file

File: hooks/useDate.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  }
+  
+  return {
+    dateValue,
+    dateValueDate,
+    dateValueHour,
+    dateFormat: dateFormat,
+    joinDateHour: joinDateHour
+  }
+}
```
Comment (fernandoamz) [inline hooks/useDate.js]: @LeyaimJimenezSoftware, the same

File: navigation/MainTabNavigator.js [hunk 5]
```diff
@@ -31,11 +34,15 @@ LoginFormStack.path = '';
...
+  {
+    headerMode: 'none'
+  }
 );
 
-EventList.navigationOptions = {
+EventStack.navigationOptions = {
   tabBarLabel: 'Lista de Eventos',
   tabBarIcon: ({ focused }) => (
     <Icon name="event-note" size={25} color={`${focused ? '#020262': '#f2f2f2' }`} />
```
Comment (fernandoamz) [inline navigation/MainTabNavigator.js]: @LeyaimJimenezSoftware, You need to separate the styles at end of file, as good practice.

File: redux/actions/Event/index.js [hunk 6]
```diff
@@ -69,4 +74,48 @@ export const cleanALert = () => {
   return {
     type: CLEAN_EVENT_ALERT
   }
-}
```
Comment (fernandoamz) [inline redux/actions/Event/index.js]: @LeyaimJimenezSoftware , the same, add enter at end of file

File: redux/actions/types.js [hunk 7]
```diff
@@ -16,4 +16,7 @@ export const CLEAN_EVENT = 'clean_event'
 export const USER_CHANGED = 'user_changed'
 export const UDPATE_USER_SUCESS = 'update_user_sucess'
 export const UPDATE_USER_ERROR = 'update_user_error'
-export const CLEAN_EVENT_ALERT = 'clean_event_alert'
\ No newline at end of file
+export const CLEAN_EVENT_ALERT = 'clean_event_alert'
+export const GET_EVENTS = 'get_events'
+export const GET_EVENTS_SUCESS = 'get_events_success'
+export const GET_EVENTS_ERROR = 'get_events_error'
```
Comment (fernandoamz) [inline redux/actions/types.js]: @LeyaimJimenezSoftware, the same

File: navigation/MainTabNavigator.js (hunk 5 above)
Comment (LeyaimJimenezSoftware) [reply navigation/MainTabNavigator.js]: @fernandoamz after the export ?

Comment (fernandoamz) [reply navigation/MainTabNavigator.js]: @LeyaimJimenezSoftware, Yes.

Comment (LeyaimJimenezSoftware) [reply navigation/MainTabNavigator.js]: okas

File: components/EventCard.js [hunk 8]
```diff
@@ -1,16 +1,28 @@
...
       />
       <View style={{ flex: 1, backgroundColor: 'rgba(0, 0, 0, 0.5)', alignSelf: 'flex-end' }}>
         <Text style={{ color: 'white', fontSize: 20, margin: 6 }}>{name}</Text>
-        <Text style={{ color: 'white', margin: 6 }}>{price}</Text>
+        <Text style={{ color: 'white', margin: 6 }}>${price}</Text>
+        <Button
+          title="Ver más"
+          color={'rgba(52, 52, 52, 0.8)'}
+          onPress={()=> navigation.navigate('ShowEvent', { 
+            name: name,
```
Comment (luisarmandodv) [inline components/EventCard.js]:
```
onPress={()=> navigation.navigate('ShowEvent', { 
  name,
  description,
  price,
  date,
  location,
})}
```

Comment (luisarmandodv) [reply components/EventCard.js]: You could do something like this, to make it clearer ⬆️

File: navigation/MainTabNavigator.js [hunk 9]
```diff
@@ -1,113 +1,19 @@
...
-  HomeStack,
-  EventList,
-  CreateEvent,
-  ProfileStack,
-});
-
-tabNavigator.path = '';
-
-export default tabNavigator;
+}
```
Comment (luisarmandodv) [inline navigation/MainTabNavigator.js]: Add breakline

File: screens/CreateEvent.js [hunk 10]
```diff
@@ -57,15 +90,45 @@ export const CreateEvent = () => {
...
+          display="default"
+          onChange={onChange}
+        />
+      )}
       <View style={styles.buttonStyle}>
         <Button
           title="Crear Evento"
           onPress={() => onButtonPress()}
-        />
+          />
```
Comment (luisarmandodv) [inline screens/CreateEvent.js]: I think this space is not necessary

File: screens/EventList.js [hunk 11]
```diff
@@ -1,44 +1,36 @@
...
-				image={evt["Image"]} 
+			{listEvents.map((evt, i) => <EventCard 
+				key={i}
+				name={evt["title"]}
+				description={evt["description"]}
+				price={evt["price"]}
+				image={evt["image"]}
+				date={evt["date"]}
+				location={evt["location"]}
+				navigation={navigation}
```
Comment (luisarmandodv) [inline screens/EventList.js]: Identation here looks weird

**Pull request merged**